src/main/java/edu/ucsb/cs156/organic/controllers/ApiController.java

```diff
@@ -26,11 +28,6 @@ protected CurrentUser getCurrentUser() {
     return currentUserService.getCurrentUser();
   }
 
-  /**
```

Comment:
Why did you remove this javadoc comment?  These comments are helpful and important, and a future requirement (not yet imposed) will be that *all* functions have these comments.

So it would be helpful to restore this.